index.js

```diff
@@ -53,6 +51,7 @@ module.exports = function(source) {
 
         const existingEntitiesPromises = bemImport.parse(
             node.arguments[0].value,
+            // FIXME: we really need this context for parsing import?
```

Comment:
It's could be a problem if we use bemNaming here
I suppose we need to use bem-fs-scheme here because we are parsing name of file